### ISSUE TITLE
Should allow version element to be null in nuspec

### ISSUE THREAD

**Issue by Daniel15** [chocolatey/choco#980]
If you leave the `version` element blank in the nuspec file (ie. `<version />`), `choco pack` fails with "Version is required", even if you set the version via the `--version` flag. 

If I run `cpack --version=1.2.3`, this error should not be thrown. I want to omit the version from my nuspec as my build script runs `choco pack` with the `--version` flag, but I can't do that at the moment.

**Comment by ferventcoder** [comment]
This is a new one to me. Perhaps it requires some value, even if it is later replaced by the specified version. Can you confirm?

**Comment by Daniel15** [comment]
Yeah I just added a placeholder value (I think I used 0.0.0) to work around
the issue. It's just confusing having a version number in the nuspec if
it's always ignored.

**Comment by ferventcoder** [comment]
What if you set it like this: `<version></version>`? I'm guessing that you are getting the error due to the way it does replacement - it may expect both sides of the element to exist and not a self-closing tag.

**Comment by Daniel15** [comment]
> What if you set it like this: <version></version>? 

@ferventcoder - I just tried that, and got the same error.
